Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -432,6 +437,22 @@ String CSSCounterStyleDescriptors::systemCSSText() const
     case System::Extends:
         builder.append(makeString("extends ", m_extendsName));
         break;
+    // FIXME: Should the following be ASSERT_NOT_REACHED() ?
```

Comment:
It can be reached in theory, but no one will do it. I returned an empty string, which should do the more correct thing here (`CSSCounterStyleRule::cssText()` will avoid writing a line for `system` in this case)